.gitignore

```diff
@@ -14,4 +14,6 @@
 rnote-ui/src/config.rs
 /temp/*
 *.mo
-*.code-workspace
\ No newline at end of file
+*.code-workspace
+
+.toggletasks.json
```

Comment:
I'm using [toggletasks.nvim](https://github.com/jedrzejboczar/toggletasks.nvim) which uses this file to write your tasks. I hope it's fine otherwise I can remove this afterwards.

Reply:
sure, it can be added.